readme.txt

```diff
@@ -40,6 +40,10 @@ This is compatible with version greater than 1.9.3 gravity forms.
 
 == Changelog ==
 
+= 1.3.4 =
+* added skip on callback if status is already marked as success
+* allowing to update failed payment status when order.paid event is triggered
```

Comment:
@abdulwahidsharief , update this one 
`allowing to update failed payment status when order.paid event is triggered` --> `allowing to update payment status when order.paid event is triggered to Paid`